### PR TITLE
Keras 3 migration

### PR DESCRIPTION
**Pull request by sachinprasadhs**

### CODE REVIEW

File: examples/nlp/md/multiple_choice_task_with_transfer_learning.md [hunk 1]
```diff
@@ -572,91 +619,147 @@ def build_model():
...
 Let's checkout the model summary to have a better insight on the model.
 
 
 ```python
 model.summary()
 ```
 
+<div class="k-default-codeblock">
+```
+Model: "model"
```
Comment (fchollet) [inline examples/nlp/md/multiple_choice_task_with_transfer_learning.md:659]: This summary is the old style summary. The example was converted to Keras 3 already, and now it seems to run on Keras 2.

Comment (sachinprasadhs) [reply examples/nlp/md/multiple_choice_task_with_transfer_learning.md:659]: Surprising that it is using Keras 2, the issue reported here has some problem with the plot model rendering https://github.com/keras-team/keras-io/issues/1713, and the original PR created in keras_core does not seem to have migrated to Keras 3 https://github.com/keras-team/keras-core/pull/884

Comment (fchollet) [reply examples/nlp/md/multiple_choice_task_with_transfer_learning.md:659]: Make sure to `pip install keras --upgrade` in your environment before generating the files.

File: examples/nlp/md/multiple_choice_task_with_transfer_learning.md [hunk 2]
```diff
@@ -692,71 +916,6877 @@ history = model.fit(
...
-    print(f"❓ Sentence {i+1}:\n{question}\n")
-    print(f"✅ True Ending: {true_answer}\n   >> {row[true_answer]}\n")
-    print(f"🤖 Predicted Ending: {pred_answer}\n   >> {row[pred_answer]}\n")
-    print("-" * 90, "\n")
+<div class="k-default-codeblock">
+```
+
+```
+</div>
+  5/183 [..............................] - ETA: 43:54 - loss: 1.3896 - accuracy: 0.2000
```
Comment (fchollet) [inline examples/nlp/md/multiple_choice_task_with_transfer_learning.md:954]: That's still using Keras 2.

Make sure to install keras 3 *after* everything else is installed.

Comment (gbaned) [reply examples/nlp/md/multiple_choice_task_with_transfer_learning.md:954]: > That's still using Keras 2.
> 
> Make sure to install keras 3 _after_ everything else is installed.

Hi @sachinprasadhs Any update on this PR? Please. Thank you!
